tests/flytekit/unit/extras/tensorflow/test_native.py

```diff
@@ -0,0 +1,29 @@
+from typing import List
+
+import tensorflow as tf
+
+from flytekit import task, workflow
+
+
+@task
+def generate_model() -> tf.keras.Model:
+    inputs = tf.keras.Input(shape=(32,))
+    outputs = tf.keras.layers.Dense(1)(inputs)
+    model = tf.keras.Model(inputs, outputs)
+    return model
+
+
+@task
+def get_model_layers(model: tf.keras.Model) -> List[tf.keras.layers.Layer]:
+    return model.layers
```

Comment:
This test shouldn't use your type transformer; it should just pickle. So it doesn't make sense to have unless you add a type transformer for `tf.Module`.

Reply:
Also, I'm unsure if List[...] will work. `tf.keras.layers.Layer` should, however, work after you add `tf.Module`.